app/src/main/java/com/infomaniak/swisstransfer/ui/screen/newtransfer/upload/components/Progress.kt

```diff
@@ -65,37 +63,44 @@ private fun Percentage(uploadedSizeInBytes: () -> Long, totalSizeInBytes: Long)
         }
     }
 
-    Text(
-        text = "$percentageNoDecimals%",
-        style = SwissTransferTheme.typography.labelRegular,
-        color = SwissTransferTheme.colors.secondaryTextColor,
-    )
+    Text(text = "$percentageNoDecimals%")
 }
 
+/**
+ * Text style is provided through [LocalTextStyle] inside [Progress]
+ */
 @Composable
 private fun UploadedSize(uploadedSizeInBytes: () -> Long) {
     val context = LocalContext.current
     val humanReadableSize by remember {
         derivedStateOf { HumanReadableSizeUtils.getHumanReadableSize(context, uploadedSizeInBytes()) }
     }
 
-    Text(
-        text = humanReadableSize,
-        style = SwissTransferTheme.typography.labelRegular,
-        color = SwissTransferTheme.colors.secondaryTextColor,
-    )
+    Text(text = humanReadableSize)
 }
 
+/**
+ * Text style is provided through [LocalTextStyle] inside [Progress]
+ */
 @Composable
 private fun TotalSize(totalSizeInBytes: Long) {
     val context = LocalContext.current
     val humanReadableTotalSize by remember {
         derivedStateOf { HumanReadableSizeUtils.getHumanReadableSize(context, totalSizeInBytes) }
     }
 
-    Text(
-        text = humanReadableTotalSize,
-        style = SwissTransferTheme.typography.labelRegular,
-        color = SwissTransferTheme.colors.secondaryTextColor,
-    )
+    Text(text = humanReadableTotalSize)
+}
+
+@PreviewLightAndDark
+@Composable
+private fun Preview() {
+    SwissTransferTheme {
+        Surface {
+            Progress(
+                progressState = { UploadWorker.UploadProgressUiState.Progress(73_614L) },
+                totalSizeInBytes = 3_279_218L
```

Comment:
Comma is still missing.

Why am I doing suggestions if people don't use them ? 😢 💔

Reply:
Good call, I did miss it when applying all of your modifications manually :/